src/sidebar/store/modules/activity.js

```diff
@@ -87,10 +143,19 @@ function isLoading(state) {
 }
 
 /**
- * Return true when annotations are actively being fetched.
+ * Return `true` if `$tag` exists in the array of annotation `$tag`s that
+ * have in-flight save requests, i.e. the annotation in question is actively
+ * being saved to a remote service.
+ *
+ * @param {object} state
+ * @param {object} annotation
+ * @return {boolean}
  */
-function isFetchingAnnotations(state) {
-  return state.activity.activeAnnotationFetches > 0;
+function isSavingAnnotation(state, annotation) {
```

Comment:
I was curious about the performance of this—but didn't see a quick path to using `reselect`'s `createSelector` as I can't quite grok how that would work with an external argument (here `annotation`)...